subsys/nrf_security/Kconfig

```diff
@@ -55,6 +55,13 @@ rsource "Kconfig.psa.nordic"
 config PSA_PROMPTLESS
 	bool
 
+config SSF_V_2
+	bool
+	default y if BOARD_NRF54H20DK_NRF54H20_CPUAPP_IRON
+	prompt "temporary option until iron and SSFv2 is available"
```

Comment:
why does this have a prompt?

Reply:
It is to allow an application that doesn't need any secure services to turn them off.